modules/dhcp_native_ms/dhcp_native_ms_plugin.rb

```diff
@@ -2,6 +2,8 @@ module ::Proxy::DHCP::NativeMS
   class Plugin < ::Proxy::Provider
     plugin :dhcp_native_ms, ::Proxy::VERSION
 
+    default_settings :disable_ddns => false
```

Comment:
Current convention is that config files examples have parameters set to their default values (with comments reflecting this). I also think that ddns updates can be disabled by default (as this is preferable configuration when used with foreman). Could you set `disable_ddns` to true here?